doc/btp_cap.txt

```diff
@@ -0,0 +1,269 @@
+
+		This command is used to execute the Unicast Audio Start procedure.
+		In case of an error, the error status response will be returned.
+		In case of a success, the IUT continues processing the command
+		asynchronously.
+
+	Opcode 0x06 - Unicast Audio Update
+		Controller Index:	<controller id>
+		Command parameters: Stream Count (1 octet)
+					Update Data (varies)
+
+		where Update Data can contain one or multiple structures defined as:
+					Address_Type (1 octet)
+					Address (6 octets)
+					ASE_ID (1 octet)
+					Metadata LTVs length (1 octet)
+					Metadata LTVs (varies)
+		The number of structures is specified with the Stream Count.
+
+		Response parameters: <none>
+
+		This command is used to execute the Unicast Audio Update procedure.
+		In case of an error, the error status response will be returned.
+		In case of a success, the IUT continues processing the command
+		asynchronously.
+
+	Opcode 0x07 - Unicast Audio Stop
+		Controller Index:	<controller id>
+		Command parameters:	CIG_ID (1 octet)
+
+		Response parameters:	<none>
+
+		This command is used to execute the Unicast Audio Stop procedure.
+		In case of an error, the error status response will be returned.
+		In case of a success, the IUT continues processing the command
+		asynchronously.
+
+	Opcode 0x08 - Broadcast Source Setup Stream
+		Controller Index:	<controller id>
+		Command parameters:	Source ID (1 octet)
+					Subgroup ID (1 octet)
+					Coding Format (1 octet)
+					VID (2 octets)
+					CID (2 octets)
+					Codec Config LTVs len (1 octet)
+					Metadata LTVs len (1 octet)
+					LTVs (varies)
+		where Source ID and Subgroup ID are local identifiers.
+
+		Response parameters:	<none>
+
+		This command is used to configure the local source stream before
+		start of the source advertising.
+		In case of an error, the error status response will be returned.
+
+	Opcode 0x09 - Broadcast Source Setup Subgroup
+		Controller Index:	<controller id>
+		Command parameters:	Source ID (1 octet)
+					Subgroup ID (1 octet)
+					Coding Format (1 octet)
+					VID (2 octets)
+					CID (2 octets)
+					Codec Config LTVs len (1 octet)
+					Metadata LTVs len (1 octet)
+					LTVs (varies)
+		where Source ID and Subgroup ID are local identifiers.
+
+		Response parameters:	<none>
+
+		This command is used to configure the local source subgroup before
+		start of the source advertising.
+		In case of an error, the error status response will be returned.
+
+	Opcode 0x0a - Broadcast Source Setup
+		Controller Index:	<controller id>
+		Command parameters:	Source ID (1 octet)
+					Sdu Interval (3 octets)
+					Framing (1 octet)
+					Max SDU (2 octets)
+					Retransmission Num (1 octet)
+					Max Transport Latency (2 octets)
+					Presentation Delay (3 octets)
+					Flags (1 octet)
+					Broadcast Code (16 octet)
+		where:
+			- Source ID is a local identifier.
+			- Flags is a bit map of settings:
+				bit 0: Use encryption
+				bit 1: Use the given Broadcast Code
```

Comment:
If the encryption is requested, then Broadcast Code has to be given. So is `bit 1` needed?

Reply:
A typo, the bit was meant to be used for `BTP_CAP_BROADCAST_SOURCE_SETUP_FLAG_SUBGROUP_CODEC = 0x02`